src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md

```diff
@@ -79,41 +92,37 @@ For a complete list of supported parameters see the <a href="https://github.com/
 Properties are returned in summary format. For example:
 
 ```
-namespace name                           value
-docker    docker.engine.LOCAL.hostname   127.0.0.1
-docker    docker.engine.REMOTE.hostname  103.67.89.6
+namespace name                    value
+docker    engine.LOCAL.hostname   127.0.0.1
+docker    engine.REMOTE.hostname  103.67.89.6
 
 Total: 2
 ```
 
 If the property value that is returned is longer than 50 characters, the first fifty characters are displayed, followed by three ellipses (`...`). For example:
```

Comment:
Why do we do that ? That's very odd. I think it's a defect rather than a feature we should be talking about.

Reply:
Where does this limitation come from ?

Reply:
I am not sure I understand these comments - let's discuss.

Reply:
removed 'docker.' from name and removed the ellipses bit and example.